src/options.rs

```diff
@@ -732,8 +735,26 @@ pub fn get_app_use_regex(matches: &ArgMatches, config: &Config) -> bool {
     false
 }
 
+pub fn get_use_terminal_name(matches: &ArgMatches, config: &Config) -> Option<String> {
```

Comment:
Forgot to add - might want to rename this to something like `get_show_hostname`.